tf-modules/cumulus-rds-tf/main.tf

```diff
@@ -94,11 +95,11 @@ resource "aws_rds_cluster" "cumulus" {
   preferred_backup_window = var.backup_window
   db_subnet_group_name    = aws_db_subnet_group.default.id
   apply_immediately       = var.apply_immediately
+  storage_encrypted       = true
```

Comment:
It's worth noting - this is *required* as the default changed from v1 to v2.      👍🏻

Reply:
Correct, v2 default is storage_encrypted = false, so we need to add it.